src/app/services/bucket/qarnotBucketService.py

```diff
@@ -13,10 +14,23 @@ def connect_to_bucket(self):
         return self.bucket
 
     async def upload_file(self, bucket_key: str, file_binary: bin):
-        return await self.connect_to_bucket().add_file(file_binary, bucket_key)
+        try:
+            self.connect_to_bucket().add_file(file_binary, bucket_key)
+        except Exception as e:
+            print(e)
```

Comment:
Perhaps a warning or logging would be better for the exception?

Reply:
Like 
```python
import logging
logging.basicFongi()
logging.error(e) ? 
```

Reply:
Yup! But regarding the logger choice, I'd recommend the uvicorn logger
```
logger = logging.getLogger("uvicorn.error")
```